mkdocs.yml

```diff
@@ -25,6 +25,22 @@ theme:
   features:
     - content.code.copy
     - navigation.indexes
+  palette:
```

Comment:
<img width="283" alt="image" src="https://github.com/user-attachments/assets/60e5ddb4-cacf-4efe-acb7-ab5545548a1f">
when I click "switch to light mode", the dark mode is turned on

Reply:
I have changed the order, the default order given in the documentation seems not working properly.